File: ws-eclipse/projetos_udemy/curso_udemy/src/menor_de_tres.java
import java.util.Locale;
import java.util.Scanner;

public class menor_de_tres {
public static void main(String[] args) {
	
 Locale.setDefault(Locale.US);
 Scanner sc = new Scanner(System.in);
 
 int valor1, valor2, valor3, menor;
 
 System.out.println("Digite o primeiro valor: ");
 valor1 = sc.next();
 
 System.out.println("Digite o segundo valor : ");
 valor2 = sc.next();
 
 System.out.println("Digite o terceiro valor : ");
 valor3 = sc.next();
 
 if((valor1 < valor2) && (valor1 < valor3)){
	 menor = valor1;
	 else if((valor2 < valor1) && (valor2 < valor3)){
		 menor = valor2;
		 else{
			 menor = valor3;
		 }
	 }
 }
 
 System.out.println("O menor dos valores é " + menor);
 
}
}

File: ws-codeblocks/projetos_udemy/menor_de_tres.c
#include <stdio.h>

int main(){

int valor1, valor2, valor3, menor;

printf("Digite o primeiro valor: ");
scanf("%d", &valor1);

printf("Digite o segundo valor : ");
scanf("%d", &valor2);

printf("Digite o terceiro valor : ");
scanf("%d", &valor3);

if ((valor1< valor2) && (valor1 < valor3)){
    valor1 = &menor;
    }else if((valor2 < valor1) && (valor2 < valor3)){
        valor2 = &menor;
        }else{
            valor3 = &menor;
        }

printf("O menor dos tres valores é %d", menor);

return 0;
}
